### ISSUE TITLE
[Feature request] Add documentation for each of the parameters in README

### ISSUE THREAD

**Issue by patrykbart** [MrForExample/ComfyUI-AnimateAnyone-Evolved#38]
It would be much easier to use the tool if I could 100% understand all the parameters in the `Animate Anyone Sampler`

**Comment by dwgeneral** [comment]
Agree. I even don't know how to add a IPAdapter model to current flow and work with Animate Anyone Sampler
